hw/ip/aon_timer/data/aon_timer_testplan.hjson

```diff
@@ -68,4 +69,16 @@
       tests: ["aon_timer_stress"]
     }
   ]
+
+  covergroups: [
+    {
+      name: timer_cfg_cg
+      desc: '''
+        Includes possible values fo all registers of AON timer. For 32b registers, bins are
+        introduced in order to simplfy coverage. Maximum and minimum values are collected in
+        a seperate bin to guarantee that they are checked. Seperate bins are generated for 
+        each bit.
```

Comment:
What you've done in `aon_timer_env_cov` is to split the 32-bit range (4 billion or so) into 32 equally spaced regions. That's a reasonable thing to do, but doesn't match what you've said here about generating a bin per bit.